File: Feitos em Java/3037.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int times = leitor.nextInt();
	for(int i=0;i<times;i++){
            int quantJoao=0,quantMaria=0;
            for(int j=0;j<3;j++){
                int x = leitor.nextInt();
                int d = leitor.nextInt();
                quantJoao+=(x*d);
            }
            for(int j=0;j<3;j++){
                int x = leitor.nextInt();
                int d = leitor.nextInt();
                quantMaria+=(x*d);
            }
            if(quantJoao>quantMaria){
                System.out.println("JOAO");
            }
            else{
                System.out.println("MARIA");
            }
	}
    }
}

File: Feitos em C/3037.c
#include <stdio.h>
int main() {
	int times,i,j;
	scanf("%d",&times);
	for(i=0;i<times;i++){
		int quantJoao=0,quantMaria=0;
		for(j=0;j<3;j++){
			int x,d;
			scanf("%d %d",&x,&d);
			quantJoao+=(x*d);
		}
		for(j=0;j<3;j++){
			int x,d;
			scanf("%d %d",&x,&d);
			quantMaria+=(x*d);
		}
		if(quantJoao>quantMaria){
			printf("JOAO\n");
		}
		else{
			printf("MARIA\n");
		}
	}
    return 0;
}
